backup/__init__.py

```diff
@@ -0,0 +1,15 @@
+import dbbackup
+
+from django.core.management import call_command
+
+from fle_utils.chronograph.models import Job
+
+def setup_backup():
+	try:
+		command = call_command('dbbackup')
+		schedule_job(command, frequency="DAILY")
+	except:
```

Comment:
This will catch **all** errors, including Normal system exit (`sys.exit`) and system interrupts (`Ctrl+C`) :) Please make this catch a more specific error:

`except KeyError` should do it.